pkgs/development/libraries/glui/default.nix

```diff
@@ -1,8 +1,26 @@
-{stdenv, fetchurl, freeglut, libGLU_combined, libXmu, libXext, libX11, libXi}:
-stdenv.mkDerivation {
-  name = "glui-2.35";
-  buildInputs = [freeglut libGLU_combined libXmu libXext libX11 libXi];
+{ stdenv, fetchurl
+, freeglut
+, libGL
+, libGLU
+, libX11
+, libXext
+, libXi
+, libXmu
+}:
+
+stdenv.mkDerivation rec {
+  pname = "glui";
+  version = "2.36";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/project/glui/Source/${version}/glui-${version}.tgz";
+    sha256 = "11r7f0k5jlbl825ibhm5c6bck0fn1hbliya9x1f253ikry1mxvy1";
+  };
+
+  buildInputs = [ freeglut libGLU libGL libXmu libXext libX11 libXi ];
+
   preConfigure = ''cd src'';
```

Comment:
sourceRoot might be more appropriate as it will leave the hook available to users.

Reply:
i thought the same thing, but the dirname seems to be related to the tarball it downloads.